feature/p4rt/otg_tests/p4rt_daemon_failure_test/p4rt_daemon_failure_test.go

```diff
@@ -359,12 +358,15 @@ func TestP4RTDaemonFailure(t *testing.T) {
 	t.Logf("Stop traffic")
 	ate.OTG().StopTraffic(t)
 
-	// Verify interfaceID did not change since the last time we read it.
-	changedID, notOk := watchID.Await(t)
-	if notOk {
-		t.Errorf("FAIL: DUT changed /interfaces/interface/state/id  during p4rt process restart.  want:%q got:%q", dutPort1.ID, changedID.String())
+	// Skip check for CISCO devices that use the same process for P4RT & gNMI.
+	if dut.Vendor() != ondatra.CISCO {
```

Comment:
Should this be a deviation instead of a check `if dut.Vendor() != ondatra.CISCO` ?

Reply:
I don't think that's a deviation.

Reply:
Can you explain why this isn't a deviation? I tend to agree with Prinika here.

Reply:
In Cisco's current implementation all gRPC services are managed by the same process (emsd), and so killing any process related to grpc will kill all hosted services. 

Is it the goal of feature profiles to mandate a specific implementation (i.e. separate daemons for each service)?

Reply:
Apologies, I thought I had replied here.

The intention is not to imply any requirements here. I see this more as just "where do we store differences between implementations" -- I'd suggest that generally doing this through deviations is cleaner as it leads to a single place to store these differences, and avoids the need for there to be `if` conditions with many vendors in them through the code.

I'll approve, but would suggest to the wider team here that for the sake of maintainability we keep these in a single place.